bofire/strategies/doe_strategy.py

```diff
@@ -20,22 +32,138 @@ def __init__(
     ):
         super().__init__(data_model=data_model, **kwargs)
         self.formula = data_model.formula
+        self.data_model = data_model
+        self._partially_fixed_experiments_for_next_design = None
+
+    def tell(
```

Comment:
Hmm, for what is this `tell` needed?

Reply:
Especially as you overwrite `tell` and not just `_tell`. That's looks a bit dangerous ...